src/cljs/athens/views/left_sidebar.cljs

```diff
@@ -112,7 +112,7 @@
 
 
 (defn left-sidebar
-  []
+  [route-name]
   (let [open? (subscribe [:left-sidebar/open])
         ;; current-route (subscribe [:current-route]) ;; TODO: disabled primary button if current route == navigation button
```

Comment:
Subscribe here instead of passing `route-name` as a parameter. Both work functionally, but left-sidebar is already making a `subscribe` call, we should maintain its independence from needing parameters passed in.

Reply:
See comment on line 117